downloads/id/github-git-advanced-cheat-sheet.md

```diff
@@ -0,0 +1,67 @@
+# Buku Saku Tingkat Lanjut
+
+## Perintah penggabungan
+
+---
+
+Batalkan penggabungan
+
+`$ git merge --abort`
+
+## Perintah penyimpanan
+
+---
+
+Simpan perubahan saat ini dalam penyimpanan dengan nama tertentu
+
+`$ git stash save <stash name>`
```

Comment:
"stash name" is not a literal here; it is something the user should *understand* and then replace with a value.  It should be "nama stash" , I believe .